pytest_subtests.py

```diff
@@ -155,14 +156,24 @@ def _capturing_output(self):
                 fixture.close()
                 captured.out = out
                 captured.err = err
+    
+    @contextmanager
+    def _capturing_logs(self):
+        logging_plugin = self.request.config.pluginmanager.getplugin("logging-plugin")
+        handler = LogCaptureHandler()
+        handler.setFormatter(logging_plugin.formatter)
+        
+        captured_logs = CapturedLogs(handler)
+        with catching_logs(handler):
+            yield captured_logs
 
     @contextmanager
     def test(self, msg=None, **kwargs):
         start = time.time()
         precise_start = time.perf_counter()
         exc_info = None
 
-        with self._capturing_output() as captured:
+        with self._capturing_output() as captured, self._capturing_logs() as capturedlogs:
```

Comment:
Minor change for better readability:

```suggestion
        with self._capturing_output() as captured_output, self._capturing_logs() as captured_logs:
```